lib/main_publiccloud.pm

```diff
@@ -116,6 +116,15 @@ sub load_publiccloud_tests {
             load_publiccloud_consoletests();
             loadtest("publiccloud/ssh_interactive_end", run_args => $args);
         }
+        elsif (get_var('PUBLIC_CLOUD_CONTAINERS')) {
```

Comment:
We could get rid of this variable and use `CONTAINER_RUNTIME`. If present, we schedule container tests.

Reply:
maybe but this would require also changes in maintenance setup , while this PR is simply about adding container tests to latest test suites . So I would do what you suggesting in separate PR ...